### PR TITLE
Added github actions to un-assigned issues after 90 days of inactivity

### PR DESCRIPTION
**Pull request by za**
Issue https://github.com/OWASP/wrongsecrets/issues/1016

 

### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [ ] Additional documentation
-   [x] Something else

### Checklist:

-   [ ] All the contributions made are solely the work of me and my co-authors
-   [ ] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [ ] I added UI tests to ensure my UI changes work (when change in the overall UI, not needed if just adding a challenge)
-   [x] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

**Review by bendehaan — COMMENTED** [review]
@za Thanks for the contribution! Could you add @commjoen and me as exemptions?

File: .github/workflows/project-management.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  unassign-issues-labeled-waiting-for-contributor-after-90-days-of-inactivity:
+    name: Unassign issues labeled "Stale" after 90 days of inactivity.
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: boundfoxstudios/action-unassign-contributor-after-days-of-inactivity@main
+        with:
+          last-activity: 90 # After how many days the issue should get unassigned
+          labels: 'stale,help wanted' # Only search for issues containing this labels (comma-separated)
+          exempt-assignees: 'Octo,Cat' # Exempt some assignees from being unassigned
```
Comment (commjoen) [inline .github/workflows/project-management.yml:18]:
```suggestion
          exempt-assignees: 'Octo,Cat,bendehaan,commjoen' # Exempt some assignees from being unassigned
```

**Comment by za** [comment]
> @za Thanks for the contribution! Could you add @commjoen and me as exemptions?

Added @bendehaan

**Review by commjoen — APPROVED** [review]
Thank you @za !

**Pull request merged**